.gitignore

```diff
@@ -1,2 +1,6 @@
 *.env.*
 .htaccess
+.idea/
+sites/*/sqlconf.php
```

Comment:
@tmccormi, @aethelwulffe -- should this be ignored?

Reply:
IMO the .idea yes. For `sqlconf.php` i thought it wise and safer cause i guess we had an issue like this where someones sqlconf was merged into the codebase and cause some disruptions.  If needed to change in the future it can be done by just removing  from the gitignore  of course i think one needs to know ahead that it was ignored - thats the only bad to newbies